app/services/user.js

```diff
@@ -0,0 +1,29 @@
+const { user } = require('../models/index');
+const logger = require('../logger');
+const { databaseError, userError } = require('../errors');
+const { DB_ERROR, USER_ERROR } = require('../../config/messageError');
+
+exports.registerUser = async (name, lastName, email, password) => {
+  try {
+    const createUser = await user.create({
+      name,
+      lastName,
+      email,
+      password
+    });
+    return createUser;
+  } catch (error) {
+    logger.error(error.message);
+    throw databaseError(DB_ERROR);
+  }
+};
+
+exports.validateUser = async email => {
+  try {
+    const userExist = await user.findOne({ where: { email } });
+    if (userExist) throw userError(USER_ERROR);
```

Comment:
Remember that the services are only to call the DB or external services, so this condition should not go here, this condition could go in the controller and the name of the service could be a more consistent one such as `findUserByEmail`.